x-pack/legacy/plugins/maps/server/maps_telemetry/maps_usage_collector.js

```diff
@@ -9,7 +9,7 @@ import { TASK_ID, scheduleTask, registerMapsTelemetryTask } from './telemetry_ta
 
 export function initTelemetryCollection(server) {
   registerMapsTelemetryTask(server);
-  scheduleTask(server, server.taskManager);
+  scheduleTask(server, server.plugins.task_manager);
```

Comment:
Elsewhere we are accessing this via `server.plugins.task_manager!`, so we're assuming that it might not be present.  Here we are not.  Shouldn't we be checking if this is null?  Since task_manager is a plugin, it _could_ be disabled.

Reply:
From a quick test, if ever `task_manager` plugin is disabled, Kibana throws the following error on startup `Unmet requirement "task_manager" for plugin "maps"`.

Since both plugins `maps` and `oss_telemetry` require `task_manager` as a dependency, this code won't execute if ever `task_manager` is disabled. Appending the `!` in TypeScript was more a way to tell TypeScript to ignore the possibility of it being `undefined` because we know it will exist within the code (due to having it as a dependency).

Reply:
I would suggest opening issues for maps and oss_telemetry as they need to fix that.